apps/etterlatte-saksbehandling-ui/client/src/components/behandling/soeknadsoversikt/familieforhold/omstillingsstoenad/Sivilstand.tsx

```diff
@@ -21,60 +15,60 @@ export const Sivilstand = ({ familieforhold, avdoed }: Props) => {
+                      {ss.gyldigFraOgMed
+                        ? format(new Date(ss.gyldigFraOgMed), DatoFormat.DAG_MAANED_AAR)
+                        : ' Mangler dato'}
+                    </Table.DataCell>
+                    <Table.DataCell>
+                      {ss.relatertVedSiviltilstand === avdoed.foedselsnummer
+                        ? `${avdoed.fornavn} ${avdoed.etternavn}`
+                        : 'Annen person'}
+                    </Table.DataCell>
+                    <Table.DataCell>
+                      {ss.relatertVedSiviltilstand === avdoed.foedselsnummer
+                        ? format(new Date(avdoed.doedsdato!), DatoFormat.DAG_MAANED_AAR)
+                        : ''}
+                    </Table.DataCell>
+                  </Table.Row>
+                )
+            )
+          ) : (
             <Table.Row>
-              <Table.HeaderCell scope="col">Status</Table.HeaderCell>
-              <Table.HeaderCell scope="col">Dato</Table.HeaderCell>
-              <Table.HeaderCell scope="col">Partner</Table.HeaderCell>
-              <Table.HeaderCell scope="col">Dødsdato</Table.HeaderCell>
+              <Table.DataCell aria-colspan={4} colSpan={4} align="center">
+                Mangler data om sivilstand
+              </Table.DataCell>
             </Table.Row>
-          </Table.Header>
-          <Table.Body>
-            {sivilstand ? (
-              sivilstand.map(
-                (ss, index) =>
-                  ss && (
-                    <Table.Row key={index}>
-                      <Table.DataCell>{ss.sivilstatus}</Table.DataCell>
-                      <Table.DataCell>
-                        {ss.gyldigFraOgMed
-                          ? format(new Date(ss.gyldigFraOgMed), DatoFormat.DAG_MAANED_AAR)
-                          : ' Mangler dato'}
-                      </Table.DataCell>
-                      <Table.DataCell>
-                        {ss.relatertVedSiviltilstand === avdoed.foedselsnummer
-                          ? `${avdoed.fornavn} ${avdoed.etternavn}`
-                          : 'Annen person'}
-                      </Table.DataCell>
-                      <Table.DataCell>
-                        {ss.relatertVedSiviltilstand === avdoed.foedselsnummer
-                          ? format(new Date(avdoed.doedsdato!), DatoFormat.DAG_MAANED_AAR)
-                          : ''}
-                      </Table.DataCell>
-                    </Table.Row>
-                  )
-              )
-            ) : (
-              <Table.Row>
-                <Table.DataCell aria-colspan={4} colSpan={4} align="center">
-                  Mangler data om sivilstand
-                </Table.DataCell>
-              </Table.Row>
-            )}
-          </Table.Body>
-        </Table>
-      </TableWrapper>
-    </SivilstandWrapper>
+          )}
+        </Table.Body>
+      </SivilstandTable>
+    </VStack>
   )
 }
+
+const SivilstandTable = styled(Table)`
+  width: 67%;
+`
```

Comment:
Er dette det tablewrapper gjorde før? Jeg tenker det er noe å potensielt se på å noe bedre breakpoints (men det er nok utenfor scope for denne endringen)

Reply:
Yup, var satt til å ha sånn bredde fra før av. Sånt generelt sett burde det være en spesiell standar for hvordan vi bygger tabellene våres